logbook-json/src/test/java/org/zalando/logbook/json/JsonHttpLogFormatterTest.java

```diff
@@ -34,6 +46,27 @@
 
 final class JsonHttpLogFormatterTest {
 
+    @BeforeAll
+    static void beforeAll() {
+        Configuration.setDefaults(new Defaults() {
```

Comment:
Instead of modifying a global static variable, couldn't we pass a concrete configuration to the `with` calls down below?

Reply:
I do think that is possible. But maybe I am missing something?

Reply:
I took another look, it is not possible to pass configuration to with calls. As far as I can tell static configuration is the only way.